modules/katello_activation_key.py

```diff
@@ -163,12 +163,14 @@ def activation_key(self, name, organization, lifecycle_environment=None, content
         response = activation_key.search({'name', 'organization'})
 
         if len(response) == 0:
-            activation_key = activation_key.create()
+            if not self.check_mode():
```

Comment:
From Ansible documentation it looks like check_mode is a variable (or property) not a function.

> \# if the user is working with this module in only check mode we do not
> \# want to make any changes to the environment, just return the current
> \# state with no modifications
> if module.check_mode:
>     return result

from: [Ansible ModuleDevelopment Walkthrough](http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_general.html)

same issue below